test/hotspot/jtreg/compiler/lib/ir_framework/test/IREncodingPrinter.java

```diff
@@ -239,6 +239,35 @@ private boolean checkCPUFeature(String feature, String value) {
             return false;
         }
 
+        // Please verify new CPU features before adding them. If we allow non-existent features
+        // on this list, we will ignore tests and never execute them.
+        List<String> verifiedCPUFeatures = new ArrayList<String>( Arrays.asList(
+            // General
+            "sha3",
+            "fma",
```

Comment:
I would put this under `x86` or something.

Reply:
Please add

// Consult `CPU_FEATURE_FLAGS` in corresponding vm_version_<cpu>.hpp file to find correct cpu feature's name.